neptune/new/internal/backends/hosted_neptune_backend.py

```diff
@@ -620,6 +620,14 @@ def to_leaderboard_entry(entry) -> LeaderboardEntry:
         except HTTPNotFound:
             raise ProjectNotFound(project_id)
 
+    def get_run_url(self, run_uuid: uuid, workspace: str, project_name: str, short_id: str) -> str:
+        return "{base_url}/{workspace}/{project}/e/{run_id}".format(
+            base_url=self.get_display_address(),
+            workspace=workspace,
+            project=project_name,
+            run_id=short_id
+        )
```

Comment:
what's wrong with f-strings? ;)
```suggestion
        base_url = self.get_display_address()
        return f"{base_url}/{workspace}/{project_name}/e/{short_id}"
```